how-to/3_Processors/FlatDarkFieldNormaliser.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# -*- coding: utf-8 -*-\n",
+    "#  Copyright 2021 - 2024 United Kingdom Research and Innovation\n",
+    "#  Copyright 2021 - 2024 The University of Manchester\n",
+    "#\n",
+    "#  Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "#  you may not use this file except in compliance with the License.\n",
+    "#  You may obtain a copy of the License at\n",
+    "#\n",
+    "#      http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "#  Unless required by applicable law or agreed to in writing, software\n",
+    "#  distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "#  See the License for the specific language governing permissions and\n",
+    "#  limitations under the License.\n",
+    "#\n",
+    "#   Authored by:    Mariam Demir (UKRI-STFC)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Normalise Data Using Flat and Dark Field Projections\n",
+    "##### Detectors can contain fixed pattern noise and varying pixel-to-pixel sensitivies, this can result in artifacts in the tomogram.\n",
+    "##### To correct for detector artifacts, we can take dark and flat field images. Dark field images are taken without any signal or sample. Flat field images are taken with the signal, but no sample. The intensities in these images can be subtracted from the sample projections.\n",
+    "##### This example shows how to use the `Normaliser` processor to perform flat and dark field corrections on projections"
+   ]
```

Comment:
```suggestion
    "### Normalise Data Using Flat and Dark Field Projections\n",
    " Detectors can contain fixed pattern noise and varying pixel-to-pixel sensitivies, this can result in artifacts in the tomogram To correct for detector artifacts, we can take dark and flat field images. Dark field images are taken without any signal or sample. Flat field images are taken with the signal, but no sample. The intensities in these images can be subtracted from the sample projections.\n",
    "This example shows how to use the `Normaliser` processor to perform flat and dark field corrections on projections"
   ]
```